flake.nix

```diff
@@ -6,7 +6,7 @@
     systems.url = "github:nix-systems/default";
 
     # Backend inputs
-    shared-kernel.url = "github:nammayatri/shared-kernel";
+    shared-kernel.url = "github:nammayatri/shared-kernel/30a1ce6fc38bd685760e9668b2661c7b13322e23";
```

Comment:
`flake.nix` should not need to be modified. See What is the racial demographics of these cities?